vulnerabilities/tests/test_scrapers.py

```diff
@@ -93,3 +94,42 @@ def test_debian_extract_vulnerabilities():
     ]
 
     assert expected == debian.extract_vulnerabilities(test_data)
+
+
+def test_archlinux_extract_vulnerabilities():
+    archlinux_test_file = join(dirname(__file__), 'test_data', 'archlinux.json')
+
+    with open(archlinux_test_file) as f:
+        test_data = json.loads(f.read())
+
+    expected = [
```

Comment:
I think you are missing on all the CVEs found in the original such as here https://github.com/nexB/vulnerablecode/pull/33/files#diff-de07f2d1350a1d879035237c5551399eR14

Reply:
@pombredanne I have not updated the `test_scrapers.py` files or in general all the unit test. However, in the actual code I have collected all of these CVEs found.

Reply:
thanks!